aboutlibraries-core/src/commonMain/kotlin/com/mikepenz/aboutlibraries/Libs.kt

```diff
@@ -26,13 +27,23 @@ data class Libs constructor(
             return this
         }
 
+        /**
+         * Don't crash on missing library data. Instead fill in:
+         * [Library.name] = use [Library.uniqueId]
+         * [Library.developers] = use [emptyList]
+         */
+        fun recoverableMissingData(recoverable: Boolean): Builder {
```

Comment:
Please drop this function, I don't think it's required to have this behaviour configured.

Reply:
removing :)